public/components/event_analytics/explorer/query_assist/input.tsx

```diff
@@ -155,6 +125,43 @@ export const QueryAssistInput: React.FC<React.PropsWithChildren<Props>> = (props
   // below is only used for url redirection
   const [autoRun, setAutoRun] = useState(false);
   const [callOut, setCallOut] = useState<React.ReactNode>(null);
+  const dismissCallOut = () => setCallOut(null);
+
+  const prohibitedQueryCallOut = (
+    <EuiCallOut
+      data-test-subj="query-assist-guard-callout"
+      title="I am unable to respond to this query. Try another question."
+      size="s"
+      color="danger"
+      iconType="alert"
+      dismissible
+      onDismiss={dismissCallOut}
+    />
+  );
+
+  const emptyQueryCallOut = (
+    <EuiCallOut
+      data-test-subj="query-assist-empty-callout"
+      title="Enter a natural language question to automatically generate a query to view results."
+      size="s"
+      color="warning"
+      iconType="iInCircle"
+      dismissible
+      onDismiss={dismissCallOut}
+    />
+  );
+
+  const pplGenerated = (
```

Comment:
Would these callouts be better placed as their own components or in some list? Slightly concerned about adding more complexity to a method that's already really long/has a lot of logic.

Reply:
thanks that's a good idea, i've made some refactors